Modules/Sources/BuildSettingsKit/BuildSettings+Live.swift

```diff
@@ -12,6 +12,7 @@ extension BuildSettings {
         eventNamePrefix = bundle.infoValue(forKey: "WPEventNamePrefix")
         explatPlatform = bundle.infoValue(forKey: "WPExplatPlatform")
         itunesAppID = bundle.infoValue(forKey: "WPItunesAppID")
+        appURLScheme = bundle.infoValue(forKey: "WPAppURLScheme")
```

Comment:
I happened to add the exact same property in https://github.com/wordpress-mobile/WordPress-iOS/pull/24260 😄

Reply:
wow even down to the naming 😮 
I haven't seen the PR – came up with this independently.